src/components/ChatLog.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+const ChatLog = ({ entries, onLikeChange }) => {
+return (
+<div className="chat-log">
+    {entries.map((entry) => (
+    <ChatEntry
+        key={entry.id}
+        id={entry.id}
+        sender={entry.sender}
+        body={entry.body}
+        timeStamp={entry.timeStamp}
+        onLikeChange={onLikeChange}
+    />
+    ))}
+</div>
+);
+};
```

Comment:
Although JS isn't blocked by whitespace like Python, we want to be mindful of proper indentation for readability! 

I'm curious what the propTypes for `ChatLog` might look like! 👀